File: code/packages/csharp/toml-parser/tests/CodingAdventures.TomlParser.Tests/TomlParserTests.cs
namespace CodingAdventures.TomlParser.Tests;

public sealed class TomlParserTests
{
    [Fact]
    public void PingReturnsPackageName()
    {
        Assert.Equal("toml-parser", new CodingAdventures.TomlParser.TomlParser().Ping());
    }
}

File: code/packages/fsharp/toml-parser/tests/CodingAdventures.TomlParser.Tests/TomlParserTests.fs
namespace CodingAdventures.TomlParser.Tests

open CodingAdventures.TomlParser.FSharp
open Xunit

module TomlParserTests =
    [<Fact>]
    let pingReturnsPackageName () =
        Assert.Equal("toml-parser", TomlParser().Ping())